test/datapipes/test_climate.py

```diff
@@ -46,34 +54,55 @@ def geopotential_filename():
     return path
 
 
+# default keyword args for DRY
+spec_kwargs = dict(
+    name="era5",
+    file_type="hdf5",
+    channels=None,
+    use_cos_zenith=True,
+    num_steps=1,
+    stride=1,
+)
+datapipe_kwargs = dict(
+    dt=1,
+    start_year=2018,
+    patch_size=8,
+    num_samples_per_year=None,
+    batch_size=1,
+    use_latlon=True,
+    num_workers=1,
+    shuffle=False,
+)
+
+
 @nfsdata_or_fail
 @import_or_fail("netCDF4")
 @pytest.mark.parametrize("device", ["cuda:0", "cpu"])
 def test_climate_hdf5_constructor(
-    data_dir, stats_dir, lsm_filename, geopotential_filename, device, pytestconfig
+    data_dir,
+    stats_files,
+    metadata_path,
+    lsm_filename,
+    geopotential_filename,
+    device,
+    pytestconfig,
 ):
 
-    from modulus.experimental.datapipes.climate import ClimateHDF5Datapipe
+    from modulus.datapipes.climate import ClimateDatapipe, ClimateDataSourceSpec
 
     # construct data pipe
-    datapipe = ClimateHDF5Datapipe(
+    spec = ClimateDataSourceSpec(
         data_dir=data_dir,
-        stats_dir=stats_dir,
-        channels=None,
-        stride=1,
-        dt=1,
-        start_year=2018,
-        num_steps=1,
-        patch_size=8,
-        num_samples_per_year=None,
-        batch_size=1,
+        stats_files=stats_files,
+        metadata_path=metadata_path,
+        **spec_kwargs
+    )
+    datapipe = ClimateDatapipe(
+        [spec],
         lsm_filename=lsm_filename,
```

Comment:
lsm_filename does not exist, same with geo potential, need to update all tests here.